### ISSUE TITLE
Is AI compiler part open source?

### ISSUE THREAD

**Issue by Canvas-Atomer** [Xilinx/Vitis-AI#1286]
I cannot find the source code of AI compiler in src folder. Is it not open source?

**Comment by quentonh** [comment]
@Canvas-Atomer No, it is not currently open source.  We are considering opening it up in the future to enable users to build their own custom operators using the DPU ISA, but not final decision has been made, and we may or may not do this.  Tell us more about your requirement so that we can incorporate your feedback into this decision making process.

**Comment by akuzminski** [comment]
This is also something that my team would want. Currently we must use vitis containers to do quantization and export of xmodels. This is sometimes tedious because we constantly install our python packages which are not present in container. If the compiler would be open source, we could just install it locally or construct our custom docker image with all the necessary tools for training, pruning, quantization and compilation.

**Comment by quentonh** [comment]
@akuzminski @Canvas-Atomer setting aside the source-code discussion for a moment, for both of your use cases would installation via shell scripts or pip install packages be helpful, and if so, do you have a preference for one or the other?

**Comment by akuzminski** [comment]
@quentonh if I had to choose, I'd say pip install but either is fine really
